virtual_rainforest/models/soil/carbon.py

```diff
@@ -1,7 +1,7 @@
 """The ``models.soil.carbon`` module  simulates the soil carbon cycle for the Virtual
-Rainforest. At the moment only two pools are modelled, these are low molecular weight
-carbon (LMWC) and mineral associated organic matter (MAOM). More pools and their
-interactions will be added at a later date.
+Rainforest. At the moment four pools are modelled, these are low molecular weight carbon
+(LMWC), mineral associated organic matter (MAOM), microbial biomass, and particulate
+organic matter (POM).
```

Comment:
I'm assuming this is just a correction based on the existing soil module code?

Reply:
Yeah I added some updates to the soil module documentation, which I maybe should have done as a separate pull request

Reply:
Nah, I think it's fine. Just wanted to check it wasn't an accident.